pkg/distributor/distributor.go

```diff
@@ -178,7 +178,7 @@ func (cfg *Config) Validate(limits validation.Limits) error {
 		return errInvalidShardingStrategy
 	}
 
-	if cfg.ShardingStrategy == util.ShardingStrategyShuffle && limits.IngestionTenantShardSize <= 0 {
+	if cfg.ShardingStrategy == util.ShardingStrategyShuffle && limits.IngestionTenantShardSize < 0 {
 		return errInvalidTenantShardSize
```

Comment:
nit: can you update the error message `invalid tenant shard size, the value must be greater than 0`

Reply:
That's a good call.